beps/bep_0003.rst

```diff
@@ -87,70 +92,147 @@ announce
 
 info
   This maps to a dictionary, with keys described below.
+  
+``piece layers``
+  A list of strings. The list contains one string for each file in the ``file tree``.
+  Each string consists of concatenated hashes of one merkle tree layer.
+  The layer is chosen so that one hash covers ``piece length`` bytes.
+  For example if the piece size is 16KiB then the leaf hashes are used.
+  If a piece size of 128KiB is used then 3rd layer up from the leaf hashes is used.
+  Files smaller or equal to the piece size are represented by an empty string since
+  the root hash is sufficient to cover the piece.
```

Comment:
We should specify how nodes which only contain leaves beyond the end of the file are handled here. The obvious choice would be to exclude them from the list.